x-pack/plugins/cloud/public/fullstory.ts

```diff
@@ -16,6 +16,7 @@ export interface FullStoryDeps {
 
 export interface FullStoryApi {
   identify(userId: string, userVars?: Record<string, any>): void;
+  setUserVars(userVars?: Record<string, any>): void;
```

Comment:
nit: Could pull out `Record<string, any>` into a separate `UserVars` type, which could then be shared between `setUserVars` and `identify`